src/d3a/models/area/redis_external_market_connection.py

```diff
@@ -155,7 +155,7 @@ def event_market_cycle(self):
         if self.area.current_market is None:
             return
         market_event_channel = f"{self.channel_prefix}/market-events/market"
-        current_market_info = self.area.current_market.info
+        current_market_info = self.area.next_market.info
```

Comment:
Can you please explain the intuition behind it?

Reply:
yes, in the `market_info` of the external devices the `next_market` is called. 
Here, it was the current market. If you rund a device aggregator and a market aggregator in parallel, you see the difference of one slot. The `next_market` makes more sense here actually, as the client user is informed, that a new market has been started. Does it make sense?

Reply:
Okay, please rename the variable `current_market_info ` to `market_info `

Reply:
will do

Reply:
hmm, you are right. the statistics are targeted to the current market, not the next market. So we should rather use the current_market for the devices instead of changing this here in the market api, right?